src/main.cpp

```diff
@@ -28,16 +28,17 @@ int main(int argc, char* argv[])
     // it here so users can pass command line arguments
     impactx::initialization::default_init_AMReX(argc, argv);
 
-    BL_PROFILE_VAR("main()", pmain);
     {
+        BL_PROFILE("main()");
         impactx::ImpactX impactX;
         impactX.init_grids();
         impactX.initBeamDistributionFromInputs();
         impactX.initLatticeElementsFromInputs();
         impactX.evolve();
         impactX.finalize();
     }
-    BL_PROFILE_VAR_STOP(pmain);
+
+    amrex::Finalize();
```

Comment:
The `main()` is only used in executable mode, not in Python mode.

Reply:
Ah thanks